fs/flail.py

```diff
@@ -41,7 +41,7 @@ def setUp(self):
         archive.extract(self.get_data("flail-0.2.0.tar.gz"), self.workdir)
         self.build_dir = os.path.join(self.workdir, 'flail-0.2.0')
         os.chdir(self.build_dir)
-        process.system("sed -i 's/.*-lm.*/& -lm' Makefile")
+        process.system("sed -i 's/.*-lm.*/& -lm/' Makefile")
```

Comment:
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1645

Reply:
sorry for that - I thought it was applied and something got rid of it ....

Reply:
Could you remove this commit?

Reply:
done